src/main/java/com/laytonsmith/core/functions/IncludeCache.java

```diff
@@ -52,19 +62,24 @@ public static ParseTree get(File file, com.laytonsmith.core.environments.Environ
 	public static ParseTree get(File file, com.laytonsmith.core.environments.Environment env,
```

Comment:
I think it would be cleaner if a non-static `get()` method was added for both of these static getters, having the static versions call the non-static versions on the `IncludeCache` from their environment. This will not break extensions that are using this, and it allows for calling these getters directly on the `IncludeCache` instance whenever that's convenient.

Reply:
I thought about that. In practice it doesn't seem to make things that much cleaner. This method is used so infrequently anyway, so I didn't think it was worth changing.